veps/sig-compute/individual-and-multifunction-hostdevices.md

```diff
@@ -0,0 +1,164 @@
+# VEP #60: Individual and Multifunction Host Devices  
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [ ] (R) Graduation criteria filled
+
+## Overview  
+
+This proposal aims to enhance KubeVirt's host devices interface by enabling more precise targeting of PCI devices and improving support for multifunction passthrough. These changes will maintain backward compatibility while providing additional flexibility in device assignment and power management.  
+
+## Motivation  
+
+KubeVirt's existing host devices interface is designed to provide a generic model for PCI passthrough, allowing VirtualMachines (VMs) to boot anywhere in the cluster. While this abstraction is valuable, it presents challenges in specific scenarios:  
+
+1. **Precise Allocation of Individual PCI Devices**  
+   - Users cannot currently request a specific device for debugging or troubleshooting.  
+   - Malfunctioning devices cannot be easily assigned to a VM for diagnostics and potential resolution without hardware replacement.  
+
+2. **Improved Multifunction PCI Passthrough**  
+   - Some PCI devices expose multiple functions (e.g., a GPU with an integrated audio controller or a network card with multiple interfaces).  
+   - Inconsistent function allocation across different physical devices can affect performance and initialization.  
+   - Power management behavior varies between single-function and multi-function passthrough, affecting device readiness after VM boot.  
+
+## Goals  
+
+- Enable booting a VM with a specific individual PCI device (for debugging or production use cases).  
+- Allow booting a VM using an entire PCI device with all its associated functions.  
+- Support requesting different hardware revisions of a device.  
+
+## Non-Goals  
+
+- Support for live migration.  
+
+## Definition of Users  
+
+- VM owners.  
+- Cluster administrators.  
+
+## User Stories  
+
+- As a cluster administrator, I want the ability to debug a specific malfunctioning PCI device.  
+- As a cluster administrator, I want the ability to boot a VM using a particular PCI device in production.  
+- As a cluster administrator, I want the ability to boot a VM using a specific silicon revision of a hardware device.  
+- As a VM owner, I want the ability to allocate an entire PCI device, including all its functions, to my VM.  
+
+## Repositories Impacted  
+
+- [https://github.com/kubevirt/kubevirt](https://github.com/kubevirt/kubevirt)  
+- [https://github.com/harvester/pcidevices](https://github.com/harvester/pcidevices) - 3rd party controller  
+
+## Design  
```

Comment:
Can you please provide information about the required changes on the virt-handler side? How should it treat the new `groupName` field in the device controller?